dcos/dcos-ui
# [spike] DCOS-15019: Poor default for container Image

Make the default container for Single Container services to be UCR
for better user experience.

Closes https://jira.mesosphere.com/browse/DCOS-15019

## Testing
1. Go to Services tab
2. Create a new Single Container service
3. In the Service tab, click "More Settings"
4. Verify that "Universal Container Runtime (UCR)" is selected as Container Runtime

## Trade-offs
If I understand this correctly, "Container Image" is not used for UCR, so disabling it when UCR is active is worth considering. Also hiding it in the Configuration tab if the service uses UCR Container.

## Dependencies
None.

## Screenshots

### Before
![peek 2019-01-04 21-26](https://user-images.githubusercontent.com/40791275/50706795-8c1e9080-1067-11e9-8e53-312f63fd766e.gif)


### After
![peek 2019-01-04 16-34](https://user-images.githubusercontent.com/40791275/50693453-a2b0f180-103f-11e9-9d30-d6e4817b7326.gif)
